backend/.gitignore

```diff
@@ -114,3 +114,4 @@ build
 .strapi-updater.json
 
 **/documentation/**/*.json
+**/documentation/**/*.html
```

Comment:
This file is autogenerated and this file gets auto-updated many times. So Hyungho figured for other file in the line 116 that we don't really need to version control. The file in 117 is the same so I included in the gitignore so that git ignores it and we don't need to save it in git.